### PR TITLE
doc/cephadm: fixing grafana cert/key generation documentation

### PR DESCRIPTION
**Pull request by rkachach**
Fixes: https://tracker.ceph.com/issues/57769

Signed-off-by: Redouane Kachach <rkachach@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [x] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/cephadm/services/monitoring.rst [hunk 1]
```diff
@@ -408,14 +408,17 @@ Configuring SSL/TLS for Grafana
...
 A custom certificate can be configured using the following commands:
 
 .. prompt:: bash #
 
-  ceph config-key set mgr/cephadm/grafana_key -i $PWD/key.pem
-  ceph config-key set mgr/cephadm/grafana_crt -i $PWD/certificate.pem
+  ceph config-key set mgr/cephadm/{hostname}/grafana_key -i $PWD/key.pem
+  ceph config-key set mgr/cephadm/{hostname}/grafana_crt -i $PWD/certificate.pem
+
+Where `hostname` is the short hostname for the host where grafana service is deployed.
```
Comment (adk3798) [inline doc/cephadm/services/monitoring.rst:421]: Is this actually always the short hostname or is it just the hostname provided when adding the host with cephadm  that shows up in `orch host ls`?

Comment (rkachach) [reply doc/cephadm/services/monitoring.rst:421]: > Is this actually always the short hostname or is it just the hostname provided when adding the host with cephadm that shows up in `orch host ls`?

Good question, it's what we could get from `{daemon_spec.host}` could this be also an IP addr?

Comment (adk3798) [reply doc/cephadm/services/monitoring.rst:421]: I think it should be the hostname cephadm has stored. It either comes from the scheduler which just has the hostnames we pass it or from the daemon description, which should match the orch ps output.

Comment (rkachach) [reply doc/cephadm/services/monitoring.rst]: fixed

**Comment by rkachach** [comment]
> @rkachach signed-off-by check is failing because the commit only affects docs but doesn't start with "doc/"
> 
> ```
> raise AssertionError("\n".join([
>                 "The title/s of following commit/s is/are not started with 'doc', but they only touch files under 'doc/'. Please make sure the commit titles",
>                 "are started with 'doc'. See the 'Submitting Patches' guide:",
>                 "https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst#commit-title",
>                 ""] +
>                 wrong_commits
>             ))
> ```

Fixed

**Pull request merged**